README.md

```diff
@@ -31,9 +31,11 @@ that apply to that project.
 *   [TypeScript Style Guide][ts]
 *   [Vim script Style Guide][vim]
 
-This project also contains [cpplint][cpplint], a tool to assist with style guide
-compliance, and [google-c-style.el][emacs], an Emacs settings file for Google
-style.
+This project also contains [google-c-style.el][emacs], an Emacs settings file
+for Google style.
+
+We used to host the [cpplint tool](https://github.com/cpplint/cpplint), but it
```

Comment:
This is slightly misleading. Google never hosted on that github org, it always hosted cpplint.py at https://github.com/google/styleguide/gh-pages/cpplint/cpplint.py, of which https://github.com/cpplint/cpplint has been forked